Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/GeoTrackingSceneView.swift

```diff
@@ -0,0 +1,204 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ARKit
+import SwiftUI
+import ArcGIS
+
+/// A scene view that provides an augmented reality world scale experience using geo-tracking.
+public struct GeoTrackingSceneView: View {
+    /// A Boolean value indicating if the camera was initially set.
+    @Binding var initialCameraIsSet: Bool
+    /// The view model for the calibration view.
+    @ObservedObject private var calibrationViewModel: WorldScaleSceneView.CalibrationViewModel
+    /// The geo-tracking configuration for the AR session.
+    private let configuration = ARGeoTrackingConfiguration()
+    /// A Boolean value that indicates if the user is calibrating.
+    private let isCalibrating: Bool
```

Comment:
Should this property be renamed to `isCalibrationViewEnabled` for example since it will not change?

Reply:
`calibrationViewIsEnabled` reads like an assertion.`calibrationIsEnabled` may be a better name suggestion, with documentation to specify that this property controls the calibration view visibility.

Reply:
SwiftUI tends to use the terms "is presented". With that in mind, `calibrationViewIsPresented` or `calibrationIsPresented` works.